fix(aio): fix ToC styling

gkalpak opened the pull request:
There seems to have been a bad rebase of #16228 on top of #16959, which affected ToC styles from both PRs. This commit restores the horizontal line under `.h1` elements and the vertical blue bar on the left-hand side of the ToC (with the circle running along the bar to indicate the active section).

Fixes #17098.

mary-poppins commented:
The angular.io preview for b7230eb95cec856332c8422d69034f7e5edbff67 is available [here][1].

[1]: https://pr17110-b7230eb95cec856332c8422d69034f7e5edbff67.ngbuilds.io/

gkalpak committed:
fix(aio): fix ToC styling

There seems to have been a bad rebase of #16228 on top of #16959, which affected
ToC styles from both PRs. This commit restores the horizontal line under `.h1`
elements and the vertical blue bar on the left-hand side of the ToC (with the
circle running along the bar to indicate the active section).

Fixes #17098

mary-poppins commented:
The angular.io preview for 8eaad9c5f0c50e55f89ab4dac435dec39e2e31c4 is available [here][1].

[1]: https://pr17110-8eaad9c5f0c50e55f89ab4dac435dec39e2e31c4.ngbuilds.io/

Foxandxss commented:
I like it, but I don't like the guide name on the ToC, not (prominent?) enough.

mary-poppins commented:
The angular.io preview for 5ffec44a72eeaf3652797347b9b7f3fa25e71c5d is available [here][1].

[1]: https://pr17110-5ffec44a72eeaf3652797347b9b7f3fa25e71c5d.ngbuilds.io/

mary-poppins commented:
The angular.io preview for 758267446a6748cbbac332bd86721c7ecfa37860 is available [here][1].

[1]: https://pr17110-758267446a6748cbbac332bd86721c7ecfa37860.ngbuilds.io/

petebacondarwin commented:
This definitely puts the horizontal rule below the contents title back :-)

petebacondarwin commented:
Mergeable once the Travis flakes are eradicated.

gkalpak commented:
I just realized that this also brings back the embedded ToC, which had mysteriously disappeared :smiley:
